include/my_stacktrace.h

```diff
@@ -59,6 +59,24 @@ void my_set_exception_pointers(EXCEPTION_POINTERS *ep);
 void my_write_core(int sig);
 #endif
 
+/* print stack traces for all threads */
+void my_pstack();
+
+/* print stack traces for all threads before assert failures */
+#define assert_with_stack_traces(expr) \
```

Comment:
I believe this should be merged in the final form, i.e. default assert implementation changed and verified that MyRocks / not-MyRocks work with it. This is probably small amount of code changing, but more effort in testing. If that's one larger task better done later then I guess MyRocks can be patched to default assert for now.